tests/integration_tests/functional/test_cmd_line_start.py

```diff
@@ -55,6 +60,67 @@ def _add_metadata_file(test_microvm, metadata_file):
     test_microvm.metadata_file = vm_metadata_path
 
 
+def _configure_network_interface(test_microvm):
```

Comment:
Will these netns and tap be cleaned up ? We should make sure that they are deleted after the test ends or at any assert error.

Reply:
I see that the ns is removed on the jailer destructor. as far as cleanup for the tap device, I couldn't find it

Reply:
Indeed, there is no destructor for the tap device. I found it very odd, especially since the `__init__()` for `class Tap` said:

> The tap will
   stay on the host as long as the object obtained by instantiating this
   class will be in scope. Once it goes out of scope, its destructor will
   get called and the tap interface will get removed.

Indeed, there was a clean-up mechanism that was removed as part of [this commit](https://github.com/firecracker-microvm/firecracker/commit/c239013d0714bc5d8a1bbf69128a801b901b0bc0). Putting it back cause some tests to fail so how about we unblock this PR, since is a dependency for v1.0 and the clean-up problem is more of a general framework issue. I will work on adding a clean-up handle for the `Tap` as part of a separate PR. @alindima @serban300, WDYT?

Reply:
Ok, sounds good to me, but please log an issue about this. Also I think that for the netns we should do the same. Have an object and have it cleaned up at the end of scope. Instead of cleaning it up as part of the jailer destruction. Since we can use netns without the jailer as we do here.